assets/js/algolia-search.js

```diff
@@ -0,0 +1,102 @@
+const searchClient = algoliasearch(
+  "1V7DZGZJKK",
+  "0ba2c5f100ff5fd004415e4abbcf9b9c"
+);
+
+const search = instantsearch({
+  indexName: "ogp_egazettes_index",
+  searchClient,
+});
+
+// Note: Publish date is formatted as YYYY-MM-DD
+search.addWidgets([
+  instantsearch.widgets.searchBox({
+    container: "#searchbox",
+    autofocus: true,
+  }),
+  instantsearch.widgets.hits({
+    container: "#hits",
+    templates: {
+      item(hit) {
+        return `
+            <h5 class="search-results">
+            <a class="search-content" href=${
```

Comment:
nit: can the PDF icon be shown the same way as how it is shown in the datagovsg search (i.e. very similar to what #350 does).

Reply:
48233d4!